.github/ISSUE_TEMPLATE/collaboration_proposal.yml

```diff
@@ -0,0 +1,43 @@
+name: Collaboration proposal
+description: Propose a collaboration with OpenRewrite
+title: "Proposal to collaborate with <your project>"
+labels: [access]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        At OpenRewrite we invite anyone to [contribute to our projects](https://github.com/openrewrite/.github/blob/main/CONTRIBUTING.md#ways-to-contribute).
+        
+        Collaborations allow us to work closely together with other communities, and provide write access to our repositories to trusted members of those communities.
+        We already collaborate with [many organizations and individuals](https://github.com/orgs/openrewrite/repositories?q=&type=public&language=&sort=stargazers), and we're always looking for new communities to work together.
```

Comment:
I felt a bit confused by this link. I would've expected this link to take me to somewhere where we show organizations or people that we've worked with. But, instead, it just lists out all of the repositories in the `openrewrite` org.

Reply:
Yes understandable; Ideally I'd have linked it to show the technology and organizations we work with, but we don't really have an explicit listing for that unfortunately, and it's a shifting target as we onboard more. But I also did not want to make a claim without providing any way to back that up for verification. A tricky balance. What would you propose we do here?

Reply:
I was hoping you might have a better idea on that 😆 . One possibility would be to link to the contributors tab in the rewrite repo -- but that is somewhat limiting. I'm also okay if we just leave it as is as I'm unsure what else we can do.

Reply:
Hah yeah I think it's just all something that we'll have to iterate over to improve; we don't have such a page now, but we can revisit that in the future. At least now we have a structure to propose to onboard new organizations, that already helps fill that possible future page.